examples/performance/00_overview.ipynb

```diff
@@ -723,7 +722,7 @@
       "#pragma omp parallel num_threads(nthreads)\n",
       "{\n",
       "  const int tid = omp_get_thread_num();\n",
-      "  posix_memalign((void**)&pr0[tid], 64, sizeof(float[y_size + 2 + 2][z_size]));\n",
+      "  posix_memalign((void**)&pr0[tid], 64, sizeof(float[y_size + 4][z_size]));\n",
```

Comment:
this is going to create loads of conflicts in my branch, if we could reinstante the evaluate=False it wouldn't be too bad...